engine/src/player-legacy.cpp

```diff
@@ -1018,6 +1023,16 @@ Exec_stat MCPlayer::setprop(uint4 parid, Properties p, MCExecPoint &ep, Boolean
 		if (dirty)
 			setlooping((flags & F_LOOPING) != 0); //set/unset movie looping
 		break;
+
+	case P_DONT_USE_QT:
+		if (!MCU_matchflags(data, flags, F_DONTUSEQT, dirty))
```

Comment:
I'm not sure I understand the use of the flags here - we have a member variable with the value in it...